tests/download_noaa_aws.sh

```diff
@@ -25,13 +25,30 @@ if [[ "$8" == "" && -z "$GEOIPS_PACKAGES_DIR" ]]; then
 fi
 
 if [[ "$1" == "" || "$bad_command" == "1" ]]; then
-    echo "Usage: $0 <satellite> YYYY MM DD HH MN <testdata_dir> <rclone_conf>"
-    echo "    satellite: goes16, goes17, himawari8, himawari9, geokompsat"
-    echo "    testdata_dir: if 'default' or not specified, defaults to: "
+    echo "Usage: $0 <satellite> YYYY MM DD HH MN <testdata_dir> <rclone_conf> <wildcard_list>"
+    echo "    satellite:"
+    echo "        goes16"
+    echo "        goes17"
+    echo "        himawari8"
+    echo "        himawari9"
+    echo "        geokompsat"
+    echo "        noaa-20"
+    echo "        noaa-21"
+    echo "        snpp"
+    echo "        jpss"
+    echo "    testdata_dir: "
+    echo "        if 'default' or not specified, defaults to: "
     echo "       \$GEOIPS_TESTDATA_DIR/test_data_noaa_aws/data/<satellite>/<YYYYmmdd>/<HHMN>"
```

Comment:
I wonder if, in the future, this should default to `test_data_noaa_<satellite_name>`.

Reply:
We'll consider that whne we switch to Evan's python downloader.